wiki/Glossary/en.md

```diff
@@ -63,12 +63,6 @@ Placing an object closer than the beat spacing may trick the player into clickin
 
 To ensure consistent beat spacing, turn on the `Distance Snap` button on the right-hand side of the editor. You can choose a beat spacing value by holding `Alt` and sliding the selector at the top right until you reach the desired value. Note that the higher the value would allow a more difficult beatmap.
 
-### Beatmapset
-
-The collection of beatmaps that are packaged inside the `.osz` file. These beatmapsets are beatmaps that were created under the same mapper.
-
-Beatmapsets will always have the same music, sounds (if mapped), and video (if any) or storyboard (if any). However, do note that some beatmaps from a beatmapset may specifically contain a certain sounds, background images, or storyboards that are separate from the others.
-
```

Comment:
Why was this removed and not repurposed as a clarification for the now-to-be-used `Beatmap` term?

Reply:
there are two reasons for this:

- we have https://github.com/ppy/osu-wiki/tree/master/wiki/Beatmap/ for that
- the glossary in its current form (a large page) will be removed in favor of small and specific articles: #4804